cmd/soci/commands/index/list.go

```diff
@@ -84,19 +90,27 @@ var listCommand = cli.Command{
 			return nil
 		})
 
-		writer := tabwriter.NewWriter(os.Stdout, 8, 8, 4, ' ', 0)
-		writer.Write([]byte("DIGEST\tSIZE\tIMAGE REF\tPLATFORM\tORAS ARTIFACT\n"))
-		for _, ae := range artifacts {
-			imgs, _ := is.List(ctx, fmt.Sprintf("target.digest==%s", ae.ImageDigest))
-			if len(imgs) > 0 {
-				for _, img := range imgs {
-					writeArtifactEntry(writer, ae, img.Name)
+		if quiet {
+			for _, ae := range artifacts {
+				os.Stdout.Write([]byte(fmt.Sprintf("%s\n", ae.Digest)))
+			}
```

Comment:
nit: I prefer to put a return statement in these types of blocks and then remove the else entirely. I find it easier to read when there's less indentation.

Reply:
Done.